packages/elements/src/chart/index.ts

```diff
@@ -443,9 +428,21 @@ export class Chart extends BasicElement {
       return;
     }
 
+    let plugins: Plugin[] = [
+      this.createPlugin(),
+      doughnutCenterPlugin
+    ];
+    
+    if (this.config.plugins && this.config.plugins?.length > 0) {
```

Comment:
with `?`, there is no need to do turthy check on `this.config.plugins`.
```suggestion
    if (this.config.plugins?.length > 0) {
```

Reply:
I try your suggestion but typescript doesn't allow this. It may be that because of I try to using Spread syntax (...). So I updated this condition by checking it is as Array.

Reply:
that makes sense.  Let's remove `?` as shown below as plugin can't be `falsy` anymore thanks to `isArray()`

```typescript
if (Array.isArray(this.config.plugins) && this.config.plugins.length > 0) {
```

Reply:
Updated.